v2/spanner-to-sourcedb/pom.xml

```diff
@@ -100,6 +100,12 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
```

Comment:
where is this being used ?

Reply:
@bharadwaj-aditya we are using @BeforeEach to create Resource for the UT inside CassandraSourceMetadataTest etc.